src/binary/ibc-relayer/commands/start.ts

```diff
@@ -14,6 +17,55 @@ import { resolveKeyFileOption } from '../../utils/options/shared/resolve-key-fil
 import { resolveMnemonicOption } from '../../utils/options/shared/resolve-mnemonic-option';
 import { signingClient } from '../../utils/signing-client';
 
+function resolveHeights(
+  {
+    scanFromSrc,
+    scanFromDest,
+    home,
+  }: {
+    scanFromSrc: number | null;
+    scanFromDest: number | null;
+    home: string;
+  },
+  logger: Logger
+): RelayedHeights | null {
+  if (!scanFromSrc && scanFromDest) {
+    throw new InvalidOptionError(
+      `You have defined "scanFromDest" but no "scanFromSrc". Both or none "scanFromSrc" and "scanFromDest" must be present.`
```

Comment:
Actually, I would allow only one.

If both are set, return early as you did.
Otherwise, try to load defaults from the file or just `null`.
If one is set, set just those 2 variables.

But... this can wait to another PR. For now, this seems clear enough

Read the "defaults" from file, or just set to 0.
Then